pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+A friendly reminder for new contributors:
+
+# What is DCO sign-off?  Why is it required?
+
+DCO stands for [Developer's Certificate of Origin](https://github.com/git/git/blob/master/Documentation/SubmittingPatches#L304-L349).
```

Comment:
Could you use a permalink for this? `master` is changing even this file from time to time, and I think https://github.com/git/git/blob/v2.18.0/Documentation/SubmittingPatches#L304-L349 would work just as well (and would not change).

Reply:
Done.  Good call. :)

* Verified permalink change in my own fork.